plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/model/HudiReplaceCommitMetadata.java

```diff
@@ -54,13 +60,13 @@ public boolean equals(Object o)
 
         HudiReplaceCommitMetadata that = (HudiReplaceCommitMetadata) o;
 
-        return compacted.equals(that.compacted);
+        return compacted == that.compacted && partitionToReplaceFileIds.equals(that.partitionToReplaceFileIds);
```

Comment:
Replace the order of compacted and partitionToReplaceFileIds. Also, wrap after `&&`

Reply:
Done.